src/files.js

```diff
@@ -92,6 +118,7 @@ FOAM_FILES([
   { name: "foam/java/Skeleton", flags: ['java'] },
   { name: "foam/java/JavaImport", flags: ['java'] },
   { name: "foam/java/refinements", flags: ['java'] },
+  { name: "foam/java/refinements", flags: ['swift'] }, // For createChildMethod_.
```

Comment:
Maybe add TODO: Move createChildMethod_ to core

Reply:
Moved that function into its own file that both swift and java make use of.